source/mesh_refinement/lithosphere_rift.cc

```diff
@@ -105,9 +105,26 @@ namespace aspect
     template <int dim>
     void
     LithosphereRift<dim>::
-    declare_parameters (ParameterHandler &)
+    declare_parameters (ParameterHandler &prm)
     {
-
+      prm.enter_subsection ("Mesh refinement");
+      {
+        prm.enter_subsection("Lithosphere with rift");
+        {
+          prm.declare_entry ("Width of refined area along rift", "100000",
+                             Patterns::Double (0),
+                             "The width of the area that is refined along rift. "
```

Comment:
Maybe add that the area is centered around the rift axis, such that the refined distance from the center is width/2.